src/app/demos/layouts/dynamic-sidebar/dynamic-sidebar-demo.component.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  StacheNavLink
+} from '../../../public';
+
+@Component({
+  selector: 'app-dynamic-sidebar-demo',
+  templateUrl: './dynamic-sidebar-demo.component.html'
+})
+export class DynamicSidebarDemoComponent {
+
+  // NOTE: To avoid the sidebar "flashing" the default navlinks,
+  // you must provide an array with at least one item.
+  public sidebarRoutes: StacheNavLink[] = [
+    {
+      name: 'Parent link',
+      path: '/parent-1',
+      children: []
+    }
+  ];
+
+  constructor() {
+
+    // Simulate async call.
+    setTimeout(() => {
```

Comment:
is it better to do this via [`timer`](https://www.learnrxjs.io/operators/creation/timer.html) instead of `setTimeout`?